### ISSUE TITLE
LaTeX will use pre-existing "notebook.tex" file

### ISSUE THREAD

**Issue by t-makaro** [jupyter/nbconvert#750]
I noticed that if a file with the name "notebook.tex" already exists in the current directory, then the conversion to pdf will not overwrite this file (probably good) and it will use that file to produce a (possibly completely different) pdf as the net result. 

I've noticed a similar issue with using template files named "notebook.tplx" too.

**Comment by takluyver** [comment]
The intermediate tex file should always be written to a temporary directory, so that it doesn't interfere with anything in your working directory. If latex then uses `notebook.tex` from the cwd in preference to our temporary file, that is indeed a problem.

**Comment by t-makaro** [comment]
Yes, latex is preferring the wrong file.

**Comment by mpacer** [comment]
can you give a directory structure that exhibits this pattern so I can try to reproduce it?

**Comment by t-makaro** [comment]
I'm on windows 10, using nbconvert 5.3.1 and MikTex-XeTeX 2.9.6501.

A directory like:
```
test/
    analysis.ipynb
    notebook.tex
```
then running:
```
D:test> jupyter nbconvert --to pdf analysis.ipynb
```
results in the file ```analysis.pdf``` compiled from the pre-existing ```notebook.tex```.

**Comment by takluyver** [comment]
I suspect that the culprit is one of the environment variables we set, like TEXINPUTS. We do that so that if the notebook references another file in its directory, such as an image, it will be found when running latex.

Maybe if we also add the temporary directory to the search path in TEXINPUTS, we can ensure that the right file is found.

**Comment by t-makaro** [comment]
I think this *might* get fixed by #768 since the notebook.tex file that nbconvert uses will always be from the temp directory.
